_posts/2021-06-10-external-domain-migration-announcement.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Migrate to the external-domain service"
+date: June 10, 2021
+excerpt: Customers need to take action to migrate from the cdn-service and custom-domain service to the external-domain service
+---
+
+## Who is this for?
+
+This post is for all users of the cdn-route or custom-domain services.
+
+## Background
+
+The custom-domain and cdn-route services leverage Let's Encrypt to provision certificates on our users' behalf.
+Several months ago, Let's Encrypt announced that they're deprecating their v1 API. To work with their new API, we've written a replacement
+to the cdn-route and custom-domain services, the external-domain service. We now need to migrate your service instances
+to the new service to ensure their certificates can continue renewing without issue. To accomplish this, we've written internal tooling to
+migrate instances automatically without service interruption, but it does require some action on your part to intiate the migration.
+
+## What you need to do
+
+The new external-domain service uses a different method of validation with Let's Encrypt, which relies on specific DNS records being created on
+your part. To begin the migration, we need you to create 2 DNS CNAME records for each domain on your custom-domain or cdn-route instances.
+
+The first CNAME should created for the record `_acme-challenge.$DOMAIN` and should have the value
+`_acme-challenge.$DOMAIN.external-domains-production.cloud.gov`.  In other words, if you have a custom-domain service created for
+`directorate.agency.gov`, you'd want to create a CNAME record `_acme-challenge.directorate.agency.gov` with value
+`_acme-challenge.directorate.agency.gov.external-domains-production.cloud.gov`.
+
+The second CNAME should be for `$DOMAIN` and should point to `$DOMAIN.external-domains-production.cloud.gov`.  Using the `directorate.agency.gov`
+example above, this would be `directorate.agency.cloud.gov.external-domains-production.cloud.gov`.  Note that the second change here changes how
```

Comment:
Can we make this a bit more explicit/simple that changing this CNAME will NOT cause downtime for users? This gets a bit technical for a lot of folks.

Reply:
I'm a little concerned about a blanket "this is safe" statement - updating your site's is something that should be approached with caution. The target audience here is the person/people making the DNS change, so I think the technical language is appropriate (but I'd happily accept suggestions on making it clearer). 
Would a "reach out with any questions" here make you feel better about this?

Reply:
@davemcorwin I did a little wordsmithing here to try to make it more readable. LMK if you think it still needs work!